### ISSUE TITLE
Hide the "Individual Deadline Extension" link when creating a new session 

### ISSUE THREAD

**Issue by hoanglan21** [TEAMMATES/teammates#11779]
- Production environment 
- Right now when the instructor clicks the button to add a new session, they can still click on the link to make individual deadline extension before actually clicking "create feedback session". However, because the session is not created yet, when they click on the link, they will get an error message. 
<img width="1238" alt="Screen Shot 2022-05-10 at 12 54 14 AM" src="https://user-images.githubusercontent.com/46897782/167752038-7622536f-7377-41d4-92e4-ee8811e10efa.png">
<img width="1238" alt="Screen Shot 2022-05-10 at 12 54 24 AM" src="https://user-images.githubusercontent.com/46897782/167752047-98c44701-8f13-45e8-a088-20cc755290ac.png">
- Solution: hide the link so that they cannot click on it until they have created the session.

**Comment by damithc** [comment]
Thanks for bringing this up @hoanglan21 
To make UI easier to learn (i.e., to reduce 'surprises'), instead of hiding, I suggest one of these options:
a) disable the link
b) show an explanatory message if the instructor clicks the link before the session has been saved

**Comment by hoanglan21** [comment]
I think disabling the link doesn't make much sense (it will just be a text instead of a link and the users would not understand why the text is there either) so I will change to the error message to "You must create a new session before setting individual's deadline).

**Comment by damithc** [comment]
> I think disabling the link doesn't make much sense (it will just be a text instead of a link and the users would not understand why the text is there either) so I will change to the error message to "You must create a new session before setting individual's deadline).

Good point @hoanglan21 . 
Suggested message: `You must save the new session before setting individual deadline extensions`.

**Comment by samuelfangjw** [comment]
> I think disabling the link doesn't make much sense (it will just be a text instead of a link and the users would not understand why the text is there either) so I will change to the error message to "You must create a new session before setting individual's deadline).

The link should still be disabled in this case. 

Consider the case where a user clicks the link and is brought to the individual extensions page with the error message. When the user clicks the back button to go back to the session edit page, any changes to the session are lost. The user will have to fill in all the information again. This is not really ideal from a user experience perspective.

Another issue if we are being pedantic is that from the individual extensions page where the error message is shown we have no way of knowing whether the session has been saved or not. The [current proposed solution](https://github.com/TEAMMATES/teammates/pull/11780/commits/f288a7ca321e76665427718beea508a801cbce86) seems to rely on the fact that `feedbackSessionName` is not set because an error is thrown when calling API. However, the error could have been thrown from any of the observables, and for any reason. In fact, it might not even be the user's fault that an error is thrown there.

In my opinion, the ideal way to solve this would be to disable the link. We can add a tooltip to clarify any ambiguity. It would look something like this:
<img width="293" alt="Screen Shot 2022-05-14 at 2 59 28 PM" src="https://user-images.githubusercontent.com/60355570/168414774-db8e1897-df5d-499a-8cf7-0006a486c9df.png">

**Comment by damithc** [comment]
@samuelfangjw you are right about the problem caused by the error message, IF the error message is shown in a subsequent page. My assumption was it will be a pop-up message (while staying in the same page).

**Comment by hoanglan21** [comment]
Okay i will fix it like @samuelfangjw's idea
